great_expectations/expectations/core/expect_column_values_to_not_be_null.py

```diff
@@ -64,17 +64,36 @@ class ExpectColumnValuesToNotBeNull(ColumnMapExpectation):
 
     library_metadata = {
         "maturity": "production",
-        "package": "great_expectations",
         "tags": ["core expectation", "column map expectation"],
         "contributors": [
             "@great_expectations",
         ],
         "requirements": [],
+        "has_full_test_suite": True,
+        "manually_reviewed_code": True,
     }
 
     map_metric = "column_values.nonnull"
     args_keys = ("column",)
 
+    def validate_configuration(
+        self, configuration: Optional[ExpectationConfiguration]
+    ) -> bool:
+        """
+        Validates that a configuration has been set, and sets a configuration if it has yet to be set. Ensures that
+        necessary configuration arguments have been provided for the validation of the expectation.
+
+        Args:
+            configuration (OPTIONAL[ExpectationConfiguration]): \
+                An optional Expectation Configuration entry that will be used to configure the expectation
+        Returns:
+            True if the configuration has been validated successfully. Otherwise, raises an exception
```

Comment:
Really nice docstrings 🙇🏽

Reply:
Not taking credit for that since they were copy/pasted haha

Reply:
aha didn't read closely enough - hey kudos for porting everything over so nicely 👀